ethportal-api/src/types/content_value/beacon.rs

```diff
@@ -99,6 +452,19 @@ impl<'de> Deserialize<'de> for BeaconContentValue {
         if let Ok(value) = HistoricalSummariesWithProof::from_ssz_bytes(&content_bytes) {
             return Ok(Self::HistoricalSummariesWithProof(value));
         }
+        if let Ok(value) = ForkVersionedLightClientBootstrap::from_ssz_bytes(&content_bytes) {
+            return Ok(Self::LightClientBootstrap(value));
+        }
+        if let Ok(value) = LightClientUpdatesByRange::from_ssz_bytes(&content_bytes) {
+            return Ok(Self::LightClientUpdatesByRange(value));
+        }
+        if let Ok(value) = ForkVersionedLightClientOptimisticUpdate::from_ssz_bytes(&content_bytes)
+        {
```

Comment:
*nit* woah, did `rust fmt` bump the `{` here? It's making me wince to have every other block include the { at the end of the if, and only on this one to bump it to the next line...

Reply:
yep, it's cargo fmt :innocent: